processors/ttX3_hotvr.py

```diff
@@ -645,21 +660,41 @@ def jetSelection(jetDict):
 else: 
     triggers = {'ee': lambda event: event.trigger_ee_flag, 'emu': lambda event: event.trigger_emu_flag, 'mumu': lambda event: event.trigger_mumu_flag}
 
-event_reco_inputs = {
+event_reco_inputs = []
+event_reco_inputs.append({
     'inputTriggersCollection': triggers,
     'inputMuonCollection': lambda event: getattr(event, muon_collection_for_selection_and_cleaning),
     'inputElectronCollection': lambda event: getattr(event, electron_collection_for_selection_and_cleaning),
     'inputJetCollection': lambda event: event.selectedJets_nominal,
     'inputBJetCollection': lambda event: event.selectedBJets_nominal_loose,
     'inputFatJetCollection': lambda event: event.selectedFatJets_nominal,
-    'inputMETCollection': [],  # to be included!
+    'inputMETCollection': lambda event: event.met_nominal,
```

Comment:
do we need to include this collection here?

Reply:
Not really for now. This goes into the input for `EventReconstruction` module, but the module itself is not using MET collections as far as I checked. I added that just in case since we now have propagated MET from AK4 jets.